exercises/diffie-hellman/src/test/java/DiffieHellmanTest.java

```diff
@@ -33,61 +36,63 @@ public void testPrivateKeyInRange() {
     @Ignore("Remove to run test")
     @Test
     public void testPrivateKeyRandomlyGenerated() {
-        BigInteger primeP = BigInteger.valueOf(7919);
+        BigInteger prime = BigInteger.valueOf(7919);
         Set<BigInteger> privateKeys = new HashSet<>();
-        for (int i = 0; i < 5; i++) privateKeys.add(diffieHellman.privateKey(primeP));
+        for (int i = 0; i < 2; i++) {
+		privateKeys.add(diffieHellman.privateKey(prime));
+	}
```

Comment:
Since you're only doing two I don't think a loop and adding it to a set is necessary. I think it would be a bit more readable to just construct two private keys and assert that they're not equal (see the last [robot name test](https://github.com/exercism/java/blob/master/exercises/robot-name/src/test/java/RobotTest.java) for an example). That way it's more immediately obvious what you're doing without needing to know how a set works 🙂

Reply:
More clean, I totally agree 🙂